src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"github.com/getlantern/golog"
+	"io/ioutil"
+	"os"
+	"os/user"
+)
+
+var (
+	log = golog.LoggerFor("util")
+)
+
+type logger func(arg interface{})
+
+// Based on https://github.com/getlantern/lantern-go/blob/master/src/lantern/config/config.go
+
+// determineConfigDir() determines where to load the config by checking the
+// command line and defaulting to ~/.lantern.
+func DetermineConfigDir() (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		fmt.Errorf("Error location user home directory %v", err)
+		return "", err
+	}
+	return usr.HomeDir + "/.lantern", nil
```

Comment:
Maybe not a top priority, but we aren't checking if `$HOME/.lantern` is a actually a directory. Also, does the `/` work on Windows too?